src/lib/ngsi/SubscriptionId.h

```diff
@@ -47,6 +47,7 @@ typedef struct SubscriptionId
   const char*   c_str(void) const;
   bool          isEmpty(void);
   std::string   render(RequestType container,bool comma);
+  std::string   toJson(RequestType container,bool comma);
```

Comment:
Style: whitespace after ,

(I understand you take render() as reference, but it is also wrongly styled... could you please fix that line also, pls?)

Reply:
Fixed in 7480bca